SparseGrids/CMakeLists.txt

```diff
@@ -180,6 +180,10 @@ add_test(SparseGridsAPI          gridtest api)
 add_test(SparseGridsC            gridtest c)
 Tasmanian_set_test_properties(TESTS SparseGridsAcceleration SparseGridsDomain SparseGridsRefinement SparseGridsGlobal SparseGridsLocal SparseGridsWavelet SparseGridsFourier SparseGridsExceptions SparseGridsAPI SparseGridsC)
 
+if (Tasmanian_ENABLE_BLAS)
+	add_test(SparseGridsLAPACK gridtest lapack)
+    Tasmanian_set_test_properties(TESTS SparseGridsLAPACK)
```

Comment:
Is this aligned to 4 spaces like the rest of the code? We use 4 spaces to align in Tasmanian.

Reply:
Changed.